KsApi/models/graphql/adapters/Backing+BackingFragmentTests.swift

```diff
@@ -5,7 +5,8 @@ import XCTest
 final class Backing_BackingFragmentTests: XCTestCase {
   func test() {
     do {
-      let fragment = try GraphAPI.BackingFragment(jsonObject: backingDictionary())
+      let variables = ["withStoredCards": true]
```

Comment:
Think you can pass the dictionary directly in as an argument since we aren't referencing it anywhere else?

Reply:
Yea I can, no problem, but I try to keep variables declared separately because it's easier to read.